contracts/TransactionRequest.sol

```diff
@@ -51,20 +55,20 @@ contract TransactionRequest is Digger, TransactionRequestInterface {
         return txnRequest.cancel();
     }
 
-    function claim() public returns (bool) {
+    function claim() public payable returns (bool) {
         return txnRequest.claim();
     }
 
     /*
      *  Data accessor functions.
      */
-    //
-    //  TODO: figure out why returning RequestLib.serialize() isn't working.
-    //
-    function requestData() constant returns (address[6],
-                                             bool[3],
-                                             uint[15],
-                                             uint8[1]) {
+     
+    // TODO: figure out why returning RequestLib.serialize() isn't working.
```

Comment:
This TODO looks like it's no longer an issue.

Reply:
removed